components/VotingCard.jsx

```diff
@@ -0,0 +1,39 @@
+import { useState } from "react";
+import { updateArticleVotes } from "../src/api";
+import { FaRegThumbsUp, FaRegThumbsDown } from "react-icons/fa";
+
+export function VotingCard({ article_id, initialVotes, setVotes }) {
+  const [isVoting, setIsVoting] = useState(false);
+
+  const handleVote = (voteChange) => {
+    setVotes((currentVotes) => currentVotes + voteChange);
+    setIsVoting(true);
+
+    updateArticleVotes(article_id, voteChange)
+      .then((updatedArticle) => {
+        setVotes(updatedArticle.votes);
```

Comment:
setting the votes again after making a database request causes issues when you spam click the vote button (because I can click faster than the API can handle requests)